lib/config/config.go

```diff
@@ -52,24 +51,25 @@ func IsFeatureBranch(branchName string) bool {
 
 func IsPerennialBranch(branchName string) bool {
   perennialBranches := GetPerennialBranches()
-  return sort.SearchStrings(perennialBranches, branchName) < len(perennialBranches)
+  return util.DoesStringArrayContain(perennialBranches, branchName)
 }
 
 
 func HasRemoteOrigin() bool {
-  return GetRemoteOriginUrl() != ""
+  return hasRemote("origin")
```

Comment:
Do we need this helper at all anymore? Calling 'hasRemote("origin")' is readable enough here in my opinion.